chatbot.py

```diff
@@ -13,25 +13,33 @@
 from googleMapsApiModule import get_from_maps_database
 from googleMapsApiModule import geocoding
 
+import enum
+from enum import Enum, auto
```

Comment:
Some nitpicking here as well. If you're only planning to use `Enum` and `auto` from the `enum` module, the first import is unnecessary :)

Reply:
sure sir i'll change this.